js/foundation.toggler.js

```diff
@@ -59,12 +59,19 @@ class Toggler extends Plugin {
     }
 
     // Add ARIA attributes to triggers
-    var id = this.$element[0].id;
-    $(`[data-open="${id}"], [data-close="${id}"], [data-toggle="${id}"]`)
-      .attr({
-        'aria-controls': id,
-        'aria-expanded': this.$element.is(':hidden') ? false : true
+    var id = this.$element[0].id,
+        $triggers = $(`[data-open~="${id}"], [data-close~="${id}"], [data-toggle~="${id}"]`),
+        that = this;
```

Comment:
Foundation uses Babel, so you can use all ES2015 features like arrow functions.

I think this part could be cleaned a bit. I'll take care of this.